packages/hl7v2-server/docker-compose.dev.yml

```diff
@@ -0,0 +1,91 @@
+version: "3.8"
+
+name: app
+
+networks:
+  metriportNetwork:
+    name: metriportNetwork
+    driver: bridge
+
+services:
+  # These are the configurations for our Node app
+  # When Docker Compose starts this container it will automatically
+  # use the Dockerfile in the directory to configure it
+  app:
+    build:
+      context: ../../
+      dockerfile: packages/api/Dockerfile.dev
+    deploy:
+      restart_policy:
+        condition: on-failure
+    depends_on:
+      # Our app does not work without our database
+      # so this ensures our database is loaded first
+      postgres:
+        condition: service_healthy
+    environment:
+      - DB_CREDS={"password":"admin","dbname":"db","engine":"postgres","port":5432,"host":"postgres","username":"admin"}
```

Comment:
no need right? update/remove these left-overs, please

Reply:
We decided to have these around to help people get the environment running - and some devs still use Docker-based images for local development (I advocate for regular Node-based, but we chose to keep both).

Reply:
ditto: follow-up PR @lucasdellabella